app/src/main/java/edu/uc/groupProject/topten/ui/main/MainFragment.kt

```diff
@@ -19,13 +18,10 @@ import edu.uc.groupProject.topten.R
  * MainFragment class.
  *
  * Responsible for creating the view, and additionally contains Observe functionality that
- * connects to the database.
+ * observes live data coming in from MainViewModel
  */
 class MainFragment : Fragment() {
 
-    companion object {
```

Comment:
Moved this to the end of the class as mentioned in the coding convention.
 link: https://kotlinlang.org/docs/coding-conventions.html#class-layout